leapp/utils/output.py

```diff
@@ -63,6 +63,9 @@ def report_info(report_paths, log_paths, fail=False):
         sys.stdout.write(pretty_block("END OF REPORT", color=Color.bold if fail else Color.green))
         sys.stdout.write("\n")
 
+    if answerfile:
+        sys.stdout.write("Answerfile has been generated at {}\n".format(answerfile))
```

Comment:
Looks like additional new line might come handy...

```python
============================================================
                       END OF REPORT                        
============================================================

Answerfile has been generated at /var/log/leapp/answerfile
[root@ivasilev-noexperimentaltuningdemofinal ~]# 
```

Reply:
This will be changed anyway, we discussed that with @shaded-enmity that the separate "place" as printed is not good know. But I suggest to keep such cosmetic changes for the following PRs.